packages/cli/src/commands/deploy/helper.ts

```diff
@@ -85,7 +85,10 @@ export const upsertFieldPlugin: UpsertFieldPluginFunc = async (args) => {
 
   const storyblokClient = StoryblokClient({ token, scope })
 
+  lookForManifestOptions(manifest?.options)
```

Comment:
```suggestion
  printManifestOptions(manifest?.options)
```

What do you think? It's just printing out information. It's not finding and returning something from something.